woocommerce/woocommerce-gutenberg-products-block
# Add Atomic SKU block

Note this is branched off of https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2607 so https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2607 needs to be merged first.

Adds an atomic block to display the product SKU.

Fixes #2521

### Screenshots

![Edit Page ‹ Local Testing — WordPress 2020-06-04 17-10-10](https://user-images.githubusercontent.com/90977/83781370-5596dc00-a686-11ea-9cbe-69976ec07ad8.png)
![Edit Page ‹ Local Testing — WordPress 2020-06-04 17-10-00](https://user-images.githubusercontent.com/90977/83781372-57609f80-a686-11ea-8a7b-e2fbc65bb3e2.png)

### How to test the changes in this Pull Request:

1. Add a Single Product Block
2. SKU should be in the layout
3. Ensure it can be interacted with and shows the link to edit the product in the inspector
4. Ensure it displays on the frontend
